app/services/logger.js

```diff
@@ -0,0 +1,43 @@
+const bunyan = require('bunyan');
+const PrettyStream = require('bunyan-prettystream');
+const { get } = require('lodash');
+
+const Logger = require('./base_logger');
+
+const pkg = require('../../package.json');
+const config = require('../config').get('logger');
+
+const env = process.env.NODE_ENV || 'dev';
+
+const loggerOptions = {
+  name: config.name || pkg.name,
+  streams: [
+    { level: 'error', stream: process.stderr },
+    { level: 'error', path: 'logs/error.log' },
+    ...get(config, 'streams', [])
+  ],
+  serializers: bunyan.stdSerializers,
+};
+
+if (config.rotationEnabled) {
+  loggerOptions.streams.push({
+    type: 'rotating-file',
+    path: `logs/${env}.log`,
+    period: '1d',
+    count: 7
+  });
+}
```

Comment:
move it to config file

Reply:
[Done](https://github.com/inst-event-emitter/eventemitter-backend/pull/7/commits/44502c79cf19007b92f9f6739774ab39f2674789#diff-a18e55398c3a9a4054717c2ccfa889fdR11)